main.js

```diff
@@ -1,5 +1,26 @@
 var foodInput = document.getElementById("foods-input");
 var btnAdd = document.getElementById("btn-add-foods");
 var foodTable = document.getElementById("foods-table");
+var url = "http://localhost:9081/foods"
 
+//Bind Event
+btnAdd.addEventListener('click',addItem);
 
+
+//Declare Funciton
+function addItem(){
+
```

Comment:
what the fuk is that ??? . wtd are you doing ?

Reply:
i supported for u,